build.sbt

```diff
@@ -19,42 +19,41 @@
 // 
 //      
 
-val scalaLibVersion = "2.12.12"
+val scalaLibVersion = "2.13.6"
 val model3Version = "2.7.1-20210602_1321_master"
+val appVersion = a8.sbt_a8.versionStamp(file("."))
+
 
 scalacOptions in Global ++= Seq("-deprecation", "-unchecked", "-feature")
 
-resolvers in Global += "a8-repo" at Common.readRepoUrl()
+//resolvers in Global += "a8-repo" at Common.readRepoUrl()
 
-publishTo in Global := Some("a8-repo-releases" at Common.readRepoUrl())
+//publishTo in Global := sonatypePublishToBundle.value
+credentials in Global += Credentials(Path.userHome / ".sbt" / "sonatype.credentials")
 
 scalaVersion in Global := scalaLibVersion
 
 organization in Global := "a8"
 
-version in Global := a8.sbt_a8.versionStamp(file("."))
+version in Global := appVersion
 
 serverConnectionType in Global := ConnectionType.Local
 
-bloopAggregateSourceDependencies in Global := true
-
-bloopExportJarClassifiers in Global := Some(Set("sources"))
-
 
 lazy val versions =
   Common
     .jvmProject("a8-versions", file("."), "versions")
     .settings(
       libraryDependencies ++= Seq(
```

Comment:
I think we'll want to add `.settings( publish := {} )` to this because we don't need to do regular publishes anymore.

Reply:
Not sure I get why.